pkg/rosa/runtime.go

```diff
@@ -71,7 +93,7 @@ func (r *Runtime) Cleanup() {
 	}
 }
 
-// Load the cluster key provided by the user into the runtime and return it
+// GetClusterKey Load the cluster key provided by the user into the runtime and return it
```

Comment:
Edited these comments because they did not have the leading function name required by automatic docs